Makefile.PL

```diff
@@ -0,0 +1,105 @@
+use strict;
+use warnings;
+use ExtUtils::MakeMaker;
+
+my $developer = -f '.gitignore';
+ExtUtils::MakeMaker->VERSION(6.98) if $developer;
+
+my %WriteMakefileArgs = (
+    NAME                => 'Text::Glob',
+    VERSION_FROM        => 'lib/Text/Glob.pm',
+    ABSTRACT_FROM       => 'lib/Text/Glob.pm',
+    AUTHOR              => 'Richard Clamp <richardc@unixbeard.net>',
+    LICENSE             => 'perl_5',
+
+    META_MERGE => {
+        'meta-spec' => { version => 2 },
+        dynamic_config => 0,
+        resources => {
+            repository => {
+                url => 'https://github.com/richardc/perl-text-glob.git',
+                web => 'https://github.com/richardc/perl-text-glob',
+                type => 'git',
+            },
+            bugtracker => {
+                mailto => 'bug-Text-Glob@rt.cpan.org',
+                web => 'https://rt.cpan.org/Public/Dist/Display.html?Name=Text-Glob',
+            },
```

Comment:
Good to know. RT is the default if it's not set, so if you prefer something else, it's best to set it.